### PR TITLE
 Create API route(s) for creating and editing an organization 

### LINKED ISSUE #112: Create API route(s) for creating and editing an organization
Called by:
- #9

Functionality:
- Create an organization
- Edit your organization's page

Restrictions:
- None for creating an organization
- Being an administrator or an organization for editing an organization's page

### PR DESCRIPTION
**Pull request by ddhanesha**
## Summary
This PR is to perform CURD operations on Organization and Organization Members.

## Changes in detail
1) API to create Organization
2) API to create/list Organization members
3) API to update/delete Organization members
4) Added two new columns to organization member table called `role_in_organization [varchar]` and time_per_week [foreign key availability field]`
5) Code formatting to `organization/urls.py` 

As always let me know if you have any questions.
Resolves #112

### CODE REVIEW

**Comment by ddhanesha** [comment]
Its still work in progress. I have to add an API to edit/delete organization member API.

File: backend/organization/views/organization_views.py [hunk 1]
```diff
@@ -21,3 +30,126 @@ def get_serializer_class(self):
...
+    lookup_field = 'url_slug'
+
+    def get_queryset(self):
+        return Organization.objects.filter(url_slug=str(self.kwargs['url_slug']))
+
+    def perform_update(self, serializer):
+        serializer.save()
+        return serializer
+
+    def perform_destroy(self, instance):
```
Comment (positiveimpact) [inline backend/organization/views/organization_views.py:112]: For destroy I think you should definitely have 'ALL' permission. (Only the owner should be able to delete an organization)

File: backend/organization/serializers/organization.py [hunk 2]
```diff
@@ -13,3 +14,21 @@ class OrganizationMinimalSerializer(serializers.ModelSerializer):
     class Meta:
         model = Organization
         fields = ('name', 'url_slug', 'image', 'short_description', 'background_image', 'country', 'state', 'city')
+
+
+class OrganizationMemberSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = OrganizationMember
+        exclude = ('created_at', 'updated_at')
```
[the remark this answers is not in the document]
Comment (ddhanesha) [reply backend/organization/serializers/organization.py:22]: Yes I agree. forgot to change.

File: backend/organization/views/organization_views.py (hunk 1 above)
Comment (ddhanesha) [reply backend/organization/views/organization_views.py:112]: Good point. Will do that through permissions.

Comment (ddhanesha) [reply backend/organization/views/organization_views.py:112]: This has been resolved.

File: backend/organization/serializers/organization.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (ddhanesha) [reply backend/organization/serializers/organization.py]: Resolved.

**Comment by ddhanesha** [comment]
Addressed all the changes. Merging this PR.

**Pull request merged**